server/routes/induction/update/workedBeforeUpdateController.ts

```diff
@@ -51,6 +53,20 @@ export default class WorkedBeforeUpdateController extends WorkedBeforeController
 
     const updatedInduction = this.updatedInductionDtoWithHasWorkedBefore(inductionDto, workedBeforeForm)
 
+    // If the previous page was Check Your Answers, decide whether to redirect back check answers on submission
+    if (this.previousPageWasCheckYourAnswers(req)) {
+      req.session.inductionDto = updatedInduction
+      // Not worked before is selected, redirect to Check Your Answers
+      req.session.workedBeforeForm = undefined
+      if (workedBeforeForm.hasWorkedBefore === YesNoValue.NO) {
```

Comment:
If coming from the check answers page and the user selects no here we can just redirect them straight back to the check answers page.